ui/app/routes/vault/cluster/secrets/backend/configuration/index.js

```diff
@@ -124,6 +126,35 @@ export default class SecretsBackendConfigurationRoute extends Route {
     }
   }
 
```

Comment:
Tried to clean up the logic here best I could manage. But open to other suggestions if you think there's a way to make this more readable.

Reply:
great work! I think this reads a lot cleaner